client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -221,19 +222,39 @@ class EmailProvidersComparison extends React.Component {
 	renderGSuiteDetails() {
 		const { currencyCode, gSuiteProduct, translate } = this.props;
 
+		let title = translate( 'G Suite by Google' );
+		let description = translate(
+			"We've partnered with Google to offer you email, storage, docs, calendars, and more."
+		);
+		let logo = gSuiteLogo;
+		let buttonLabel = translate( 'Add G Suite' );
+
+		if ( config.isEnabled( 'google-workspace-migration' ) ) {
+			title = getGoogleMailServiceFamily();
+			description = translate(
+				'The best way to create, communicate, and collaborate. An integrated workspace that is simple and easy to use.'
+			);
+			logo = googleWorkspaceLogo;
+			buttonLabel = translate( 'Add %(googleMailService)s', {
+				args: {
+					googleMailService: getGoogleMailServiceFamily(),
+				},
+				comment: '%(googleMailService)s can be either "G Suite" or "Google Workspace"',
+			} );
+		}
+
 		return (
 			<EmailProviderDetails
-				title={ translate( 'G Suite by Google' ) }
-				description={ translate(
-					"We've partnered with Google to offer you email, storage, docs, calendars, and more."
-				) }
-				image={ { path: gSuiteLogo } }
+				title={ title }
+				description={ description }
+				image={ { path: logo } }
 				features={ [
 					translate( 'Annual billing' ),
 					translate( 'Send and receive from your custom domain' ),
 					translate( '30GB storage' ),
 					translate( 'Email, calendars, and contacts' ),
-					translate( 'Video calls, Docs, spreadsheets, and more' ),
+					translate( 'Video calls, docs, spreadsheets, and more' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.